packages/streamline/src/components/cards/card-carousel/card-carousel.types.ts

```diff
@@ -0,0 +1,47 @@
+import { CardHeaderProps } from '../../../primitives/card/card-header/card-header.types';
+import { CardProps } from '../../../primitives/card/card.types';
+import { IconsName } from '../../../primitives/icon/icon.types';
+import { ColorTheme } from '../../../theme';
+
+export type CardCarouselProps = Pick<
+  CardProps,
+  'accessibilityLabel' | 'testID' | 'onPress' | 'onPressIn' | 'onPressOut'
+> &
+  Omit<CardHeaderProps, 'colors' | 'title'> & {
+    /**
+     * Card's height
+     */
+    size?: 'SM' | 'LG';
+    /**
+     * Icon of the card
+     */
+    iconName?: IconsName;
+    /**
+     * Indicate the loading state of the card
+     */
+    isLoading?: boolean;
+    /**
+     * Background color
+     */
+    backgroundColor?: ColorTheme;
```

Comment:
We do not need to have a `backgroundColor` props. The background should be an Image! I think you forgot this point 😬

Reply:
Changed